fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -2904,25 +2904,25 @@ A currency conversion quote is irrevocable; it cannot be changed after it has be
 
 #### 6.12.1 Resource version history
 
-Table 48 contains a description of each different version of the **/fxquotes** resource.
+Table 48 contains a description of each different version of the **/fxQuotes** resource.
 
 | Version | Date | Description|
 | ---- | ---- | ---- |
 |**2.0**|2024-02-26|Initial version|
 
-**Table 48 –- Version history for resource /fxquotes**
+**Table 48 –- Version history for resource /fxQuotes**
 
 #### 6.12.2 Service Details
 
 [Figure FX1](#figure-fx1) shows how the currency quotation process works, using the **POST /fxQuotes** service.
 
-Currency conversion is a process that takes place between two FSPs: the requesting DFSP and the FXP. Although the success or failure of a currency conversion may depend on the outcome of a related payment, the conversion itself is a transaction between the two FSPs. The DFSP is asking the FXP to provide cover for the payment it intends to make. The conversion does not, therefore, require KYC information to support it.
+Currency conversion is a process that takes place between two FSPs: the requesting FSP and the FXP. Although the success or failure of a currency conversion may depend on the outcome of a related payment, the conversion itself is a transaction between the two FSPs. The FSP is asking the FXP to provide cover for the payment it intends to make. The conversion does not, therefore, require KYC information to support it.
 
-Either the payer DFSP or the payee DFSP may request currency conversion. If the payer DFSP requests conversion, then the transfer itself will be denominated in the target currency and the payee DFSP need know nothing about the fact that the conversion was performed: as far as it is concerned, the transfer is in the currency they expect. Conversely, if the payee DFSP is undertaking the conversion, then the transfer will be denominated in the source currency.
+Either the payer FSP or the payee FSP may request currency conversion. If the payer FSP requests conversion, then the transfer itself will be denominated in the target currency and the payee FSP need know nothing about the fact that the conversion was performed: as far as it is concerned, the transfer is in the currency they expect. Conversely, if the payee FSP is undertaking the conversion, then the transfer will be denominated in the source currency.
```

Comment:
"payer FSP" to "Payer FSP", "payee FSP" to "Payee FSP" to align with elsewhere in the document..